docs/3.0.0-beta.x/getting-started/introduction.md

```diff
@@ -1,78 +1,77 @@
+### Custom Content Structure
 
 You can generate the admin panel in a few clicks and get your whole CMS setup in a few minutes.
 
-### Manage content
 
-The Strapi's admin panel gives you an intuitive interface to create edit and delete your content.
+### Easy Content Management
 
-### Developer-friendly API
+Strapi's admin panel gives you an intuitive interface to create, edit and delete your content.
 
-Your content have to be displayed on your devices, an API is required. Strapi provide you an API that match your need. Fetch the data you want via a REST API or via the GraphQL endpoint.
+### Developer-Friendly API
 
-### Roles & permissions
+If your content has to be displayed to the world, an API is required. Strapi provides you with an API that will easily match your needs. Fetch any data you might need via a REST API or GraphQL endpoint.
 
-A User system is built in Strapi, that let you manage how can access to what.
+### Roles & Permissions
+
+Strapi has a built-in user system that allows you to manage who can access what.
 
 ### Plugin system
 
-You want a new feature in you application, fetch a plugin in the market place and tada! You can install a GraphQL endpoint in one command.
+Need a new feature for your application? Fetch a plugin in the marketplace and tadah! Installing a GraphQL endpoint has never been so easy.
 
 ### Customization
 
-You can customize every single pice of your application. The architecture of Strapi let you customize your app to match your exact needs.
+Every part of your application can be easily customized. Strapi's architecture allows you to extend your app to match your exact use case.
 
-Don't stay stuck by the techno you choose.
+Don't be bogged down by tech you don't need!
 
 ## The Strapi Ecosystem
 
-### Code Source
+### Source Code
 
-Strapi is an Open Source project, all the source code is under the MIT licence. The core project is in the [strapi/strapi](https://github.com/strapi/strapi) GitHub repository and you will find all other tools into the [Strapi](https://github.com/strapi) GitHub organization.
+Strapi is an open-source project, all of Strapi's source code is under the MIT license. The core project can be found at the [strapi/strapi](https://github.com/strapi/strapi) GitHub repository and you will find all other tools in the [Strapi](https://github.com/strapi) GitHub organization.
 
 ### Marketplace
 
-In the marketplace section of the admin panel, you will only found official Strapi plugins. For now, the Strapi marketplace is not open to everyone but will be soon, stay tuned.
+Official Strapi plugins will be found in the marketplace section of the admin panel. Unfortunately, the Strapi marketplace is not open to outside contribution, but stay tuned as that may change in the near future.
 
-But you already can develop yours and publish them on npm.
+With that said you can still [develop & use your own plugins.](https://strapi.io/documentation/3.0.0-alpha.x/plugin-development/quick-start.html)
```

Comment:
Can you update this link to point to the beta instead of the alpha version of the doc :)

Reply:
Oops my bad haha, will do :)